functions

```diff
@@ -76,3 +76,10 @@ function log_summary()
     echo "$($TIME_STAMP) " "$@"
 }
 
+# @brief log audit message
+# @param message
+function log_audit()
```

Comment:
Wrong place for this. The `functions` file is a replacement for `phosphor-debug-collector` to work as `health logs` command. If you want to make some function universally available to all `obmc-yadro-cli` scripts, simply add it to `clicmd`.